glitch/tests/parser/terraform/files/comments.tf

```diff
@@ -0,0 +1,16 @@
+#comment1
+//comment2
+resource "google_bigquery_dataset" "dataset" {
```

Comment:
To keep the consistency to the style conventions of Terraform this file should  also have 2 space identation